### ISSUE TITLE
Better request errors in replay

### ISSUE THREAD

**Issue by sw33tLie** [caido/caido#719]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Description

When a request fails in the repeater, you get a (No response found) generic error.
This should be more verbose and tell exactly why the request failed (DNS error, server closed the connection, etc...).

**Comment by Sytten** [comment]
Agreed, this will be part of the replay refactor. Not a priority right now for us, but I am collecting the requirements for the pass we make on it.

**Comment by Sytten** [comment]
We fixed that a while back
